data/net/minecraft/world/entity/ai/gossip/GossipContainer.mapping

```diff
@@ -16,6 +38,15 @@ CLASS net/minecraft/world/entity/ai/gossip/GossipContainer
 		COMMENT Returns the greater of two int values
 		ARG 0 value1
 		ARG 1 value2
+	METHOD remove (Ljava/util/UUID;Lnet/minecraft/world/entity/ai/gossip/GossipType;)V
+		ARG 1 uuid
+		ARG 2 gossipType
+	METHOD remove (Ljava/util/UUID;Lnet/minecraft/world/entity/ai/gossip/GossipType;I)V
+		ARG 1 uuid
```

Comment:
The `uuid` here either needs a better name or a Javadoc explaining what this UUID is.